shpkpr/marathon/client.py

```diff
@@ -31,27 +31,25 @@ def _make_request(self, method, path, **kwargs):
         request = getattr(requests, method.lower())
         return request(self._build_url(path), **kwargs)
 
-    @property
-    def embed_params(self):
+    def embed_params(self, entity_type):
         return [
-            "app.tasks",
-            "app.counts",
-            "app.deployments",
-            "app.lastTaskFailure",
-            "app.taskStats",
+            "{0}.tasks".format(entity_type),
```

Comment:
It's a string formatting placeholder, treated as a normal string unless the `.format` method is called.

Python has 2 methods of string formatting:
- Plain old `"Hello %s" % "Paul"`
- Or the newer `.format` which is more powerful:
  - `"Hello {0}".format("Paul")` (the number controls the index of the argument that is used to format)
  - `"Hello {name}".format({"name": "Paul"})` (named placeholders with a dict/hash)